src/transformers/pipelines/visual_question_answering.py

```diff
@@ -123,9 +123,9 @@ def preprocess(self, inputs, padding=False, truncation=False, timeout=None):
         model_inputs.update(image_features)
         return model_inputs
 
-    def _forward(self, model_inputs):
+    def _forward(self, model_inputs, **generate_kwargs):
```

Comment:
Context for others: this is how we do it in other generative pipelines, e.g. [here](https://github.com/huggingface/transformers/blob/ece1b62b93cde70233f235f6a4c84e37bfc8eba0/src/transformers/pipelines/text_generation.py#L297)